new_game.py

```diff
@@ -0,0 +1,139 @@
+# Python 2
+from PyDictionary import PyDictionary
+import random, time, os
+
+class Data(object):
+
+    # Builds the word database
+    def collect_words(word_file="gre_word_file.txt"):
+        wordlist = open(word_file).read().splitlines()
+        return wordlist
+
+    def __init__(self, words=collect_words()):
```

Comment:
why are you doing collect_words() in the parameter? This is also bad to have a mutable object as a parameter. The next time you run this, you might end up with the same objects. Plus you are not calling self.collect_words(). just collect_words() would try to look for a collect_words method outside of your class (which doesn't exist)

You should do this:
```
def __init__(self, words=None):
    if not words:
        self.word = self.collect_words()
    else:
         self.word = word
```